kong/db/schema/typedefs.lua

```diff
@@ -174,11 +188,15 @@ typedefs.ip = Schema.define {
 }
 
 
-typedefs.cidr = Schema.define {
+typedefs.ip_or_cidr = Schema.define {
   type = "string",
-  custom_validator = validate_cidr,
+  custom_validator = validate_ip_or_cidr,
 }
 
+typedefs.cidr_v4 = Schema.define {
```

Comment:
could you add 
```
-- deprecated alias
typedefs.cidr = typedefs.cidr_v4
```
to support it as a deprecated alias? just in case a third-party plugin uses the typedef with the old name.

Reply:
Added.